interface/src/app/gene/network/network.js

```diff
@@ -187,20 +205,22 @@ angular.module('adage.gene.network', [
           NodeService.get(
             {'heavy_genes': heavyGenes, 'limit': 0},
             function success(response) {
-              var n = response.objects.length;
+              var i = 0, n = response.objects.length;
+              var uri;
```

Comment:
Definitely a step in the "more descriptive" direction, but `html` only describes the *format*, not the logical *function* of the variable. I guess a fully-verbose description would be the "HTML-formatted text of the tip to be displayed upon clicking a node." Maybe "nodeTipHTML" would be a succinct way of saying that. But this is a minor quibble so it's up to you whether you think it's worth fixing.